python/cugraph/graph/test_graph.py

```diff
@@ -268,3 +271,37 @@ def test_degree_functionality(graph_file):
     assert err_in_degree == 0
     assert err_out_degree == 0
     assert err_degree == 0
+
+
+def test_renumber():
+    source_list = ['192.168.1.1',
```

Comment:
That's a pretty cool test. 

I would just recommend adding another python test with more than 1025 vertices. The reason is that smaller tests will run in only one block in the GPU. Many problems can remain silent if we test just that. I noticed the 100k test on the C++ side but only python tests are being run in Jenkins currently. We use `netscience` dataset at the python level to cover this. I was thinking you could apply a simple transformation to it, call the renumbering, and check that it corresponds to the original dataset.